Comp2023/src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -120,13 +132,13 @@ public Arm( )
     // SmartDashboard.putNumber("CL_pidKi", m_pidKi);
     // SmartDashboard.putNumber("CL_pidKd", m_pidKd);
 
-    // SmartDashboard.putNumber("CL_stowHeight", m_stowHeight);
+    // SmartDashboard.putNumber("CL_stowAngle", m_stowAngle);
```

Comment:
All of these names should have the subsystem prefix on them (not CL) -- probably AR for arm.